demo/skeleton/template_ems/dashboard/json_menu_nested.twig

```diff
@@ -1,4 +1,4 @@
-{% extends '@EMSCore/components/json_menu_nested/template.twig' %}
+{% extends '@EMSAdminUI/bootstrap5/components/json_menu_nested/template.twig' %}
```

Comment:
Maybe introduce a twig global constant? LIke the one to format dates?

Reply:
That's a good Idea

Reply:
https://github.com/ems-project/elasticms/issues/991